### PR TITLE
[BD-46] feat: added CSS custom media query support

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
-  Ensure we can generate `@custom-media` from design tokens with style-dictionary.
-  Ensure we add support for `postcss-custom-media` to @edx/frontend-build's default Webpack configs for all MFEs.

Issue: https://github.com/openedx/paragon/issues/2015

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang!


When this pull request is ready, tag your edX technical lead.

File: config/webpack.dev.config.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const path = require('path');
 const PostCssAutoprefixerPlugin = require('autoprefixer');
 const PostCssRTLCSS = require('postcss-rtlcss');
+const CustomMediaCSS = require('postcss-custom-media');
```
Comment (adamstankiewicz) [inline config/webpack.dev.config.js:12]: nit: might recommend including `PostCss` as a prefix to be consistent with these other PostCSS plugin imports above.

File: config/webpack.dev.config.js [hunk 2]
```diff
@@ -85,6 +86,7 @@ module.exports = merge(commonConfig, {
                 plugins: [
                   PostCssAutoprefixerPlugin(),
                   PostCssRTLCSS(),
+                  CustomMediaCSS(),
```
Comment (adamstankiewicz) [inline config/webpack.dev.config.js:89]: We'll need to also make similar changes to the other Webpack configs as well, i.e.:
* `webpack.dev-stage.config.js`
* `webpack.prod.config.js`

**Review by adamstankiewicz — APPROVED** [review]
LGTM! This addition is a great feature to have in frontend-build :)

**Comment by openedx-webhooks** [comment]
@PKulkoRaccoonGang 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/frontend-build&entry.867055334=https://github.com/openedx/frontend-build/pull/306&entry.1484655318=https://github.com/PKulkoRaccoonGang&entry.752974735=2023-02-19+23:33&entry.1917517419=2023-02-24+12:38&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**